knowledge/science/botany/jasminum/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: Samy Zemmouri
+domain: flowers
+seed_examples:
+- answer: The English common name of "Arabian jasmine", Jasminum sambac is due to it being widely cultivated in Southwest of Saudi Arabia, Yemen and Oman|
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!